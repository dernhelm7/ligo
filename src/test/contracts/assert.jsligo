let main = ([p, s] : [bool, unit]) => {
  let u : unit = assert (p);
  return (([] as list<operation>), s);
};

let some = (o : option<unit>) => {
  assert_some(o)
};
