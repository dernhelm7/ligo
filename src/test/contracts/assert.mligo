let main (p, s : bool * unit) =
  let () : unit = assert p
  in ([] : operation list), s

let some (o : unit option) =
  assert_some o
