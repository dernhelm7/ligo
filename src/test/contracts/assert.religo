let main = (p, s : bool, unit) => {
  let u : unit = assert (p);
  ([]: list (operation), s);
};

let some = (o : unit option) => {
  assert_some o
};
