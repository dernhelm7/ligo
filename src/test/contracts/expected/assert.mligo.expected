let main (p, s : bool * unit) =
  let u : unit = assert p in
  ([] : operation list), s

let some (o : unit option) = assert_some o
